packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -293,13 +294,16 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
     }
   }
 
-  @autobind
-  private _onRenderLabel(props: ITextFieldProps): JSX.Element | null {
-    const {
-      label
-    } = props;
-    if (label) {
-      return (<Label htmlFor={ this._id }>{ label }</Label>);
+  private _onRenderLabel = (props: ITextFieldProps): JSX.Element | null => {
+    if (props.label) {
+      return (<Label htmlFor={ this._id }>{ props.label }</Label>);
+    }
+    return null;
+  }
+
+  private _onRenderDescription = (props: ITextFieldProps): JSX.Element | null => {
+    if (props.description) {
```

Comment:
could we update this to be able to call the `onRenderDescription` prop even if the `description` prop is not provided? i think for our use case we'd like our wrapping TextField component to provide `onRenderDescription` without providing `description` since our description will allow JSX and cannot be passed in to the Fabric description

Reply:
ah i see you already implemented this change within `render`